### ISSUE TITLE
NVDA Installation fails on German Windows in some cases

### ISSUE THREAD

**Issue by winman3000** [nvaccess/nvda#6059]
If you want to install NVDA on a Windows Server Operating System like Windows Server 2008 R2 or Windows SBS 2011, you get an error message wich tells you that the Installation was not successfull.

**Comment by josephsl** [comment]
Hi, which server variant is this – GUI or Server Core? Note that in order for NVDA to function under server operating systems, a full GUI mode and WoW64 must be present. Thanks.

**Comment by winman3000** [comment]
Here is the log file.

INFO - __main__ (20:49:28):
Starting NVDA
INFO - core.main (20:49:30):
Config dir: C:\Users\Administrator\AppData\Roaming\nvda
INFO - core.main (20:49:30):
NVDA version 2016.1
INFO - core.main (20:49:30):
Using Windows version 6.3.9600 server
INFO - core.main (20:49:30):
Using Python version 2.7.11 (v2.7.11:6d1b6a68f775, Dec  5 2015, 20:32:19) [MSC v.1500 32 bit (Intel)]
INFO - core.main (20:49:30):
Using comtypes version 0.6.2
WARNING - tones (20:49:30):
Failed to initialize audio for tones
ERROR - synthDriverHandler.setSynth (20:49:30):
setSynth
Traceback (most recent call last):
  File "synthDriverHandler.pyo", line 86, in setSynth
  File "synthDrivers\espeak.pyo", line 39, in __init__
  File "synthDrivers\_espeak.pyo", line 289, in initialize
  File "nvwave.pyo", line 137, in __init__
  File "nvwave.pyo", line 156, in open
  File "nvwave.pyo", line 87, in _winmm_errcheck
WindowsError: [Error 2] A device ID has been used that is out of range for your system.
INFO - synthDriverHandler.setSynth (20:49:30):
Loaded synthDriver silence
INFO - core.main (20:49:30):
Using wx version 3.0.2.0 msw (classic)
INFO - braille.initialize (20:49:30):
Using liblouis version 2.6.5
INFO - braille.BrailleHandler.setDisplayByName (20:49:30):
Loaded braille display driver noBraille, current display has 0 cells.
INFO - brailleInput.initialize (20:49:30):
Braille input initialized
WARNING - core.main (20:49:30):
Java Access Bridge not available
INFO - core.main (20:49:31):
NVDA initialized
ERROR - RPC process 1184 (nvda_slave.exe) (20:50:09):
__main__.main:
slave error
Traceback (most recent call last):
  File "nvda_slave.pyw", line 29, in main
  File "installer.pyo", line 391, in install
  File "installer.pyo", line 199, in registerInstallation
  File "installer.pyo", line 51, in createShortcut
  File "comtypes\client\lazybind.pyo", line 199, in __setattr__
  File "comtypes\automation.pyo", line 664, in _invoke
COMError: (-2147352567, 'Ausnahmefehler aufgetreten.', (None, None, None, 0, None))
ERROR - gui.installerGui.doInstall (20:50:09):
Installation failed: 1
WARNING - watchdog._watcher (20:51:56):
Core frozen in stack:
  File "nvda.pyw", line 190, in <module>
  File "core.pyo", line 382, in main
  File "wx\_core.pyo", line 8657, in MainLoop
  File "wx\_core.pyo", line 7952, in MainLoop
  File "core.pyo", line 354, in Notify
  File "mouseHandler.pyo", line 144, in pumpAll
  File "mouseHandler.pyo", line 100, in executeMouseMoveEvent
  File "NVDAObjects\__init__.pyo", line 249, in objectFromPoint
  File "NVDAObjects\__init__.pyo", line 189, in findBestAPIClass
  File "NVDAObjects\__init__.pyo", line 184, in findBestAPIClass
  File "NVDAObjects\window\__init__.pyo", line 102, in getPossibleAPIClasses
  File "_UIAHandler.pyo", line 279, in isUIAWindow
  File "_UIAHandler.pyo", line 273, in _isUIAWindowHelper
  File "gui\__init__.pyo", line 473, in onActivate
  File "wx\_windows.pyo", line 2849, in PopupMenu

**Comment by jcsteh** [comment]
Is this NVDA running with a language other than English?

**Comment by winman3000** [comment]
Yes, it is a German system.

**Comment by jcsteh** [comment]
Are NVDA's messages and dialogs in German as well or are those in English?

**Comment by winman3000** [comment]
As far as I know, the installation dialogs are in German on this server.

**Comment by jcsteh** [comment]
For some reason, setting the shortcut key for the desktop shortcut fails on some German systems. The key has been translated to "Strg+Alt+N" for German, but for some reason, some German systems don't like this. I don't know what circumstances trigger this.

**Comment by jcsteh** [comment]
We're wondering whether this might have something to do with system default language versus user default language.

**Comment by bdorer** [comment]
could you try again? Please  disable the following checkbox.
Create &desktop icon and shortcut key (control+alt+n)
German: Erstelle Desktop-Verknüpfung und Tastenkürzel (STRG+Alt+N)

This bug should be fixed in newer snapshots and releases. Dave fixed the 
translation.

thanks


Am 28.06.2016 um 09:21 schrieb James Teh:

**Comment by bhavyashah** [comment]
@bdorer  states that the necessary translation revisions have been made and this issue should now be fixed. Should we therefore close as per previous comment? @ehollig

**Comment by bdorer** [comment]
I wote for closing as I never heared of such problems any more. If I 
could, I'd do.

Am 03.08.2017 um 08:18 schrieb bhavyashah:

**Comment by Adriani90** [comment]
@winman3000 can you still reproduce this issue?

**Comment by Adriani90** [comment]
We do not have updates on this issue since a very long time and I have never reproduced this issue. I don't have any user reports stating that this still occurs. The only thing I can imagine here is that the user account control (UAC) is set to 100% and blocks the installation. But this occurs with many other programs and is not an NVDA specific issue.
However, if you still have this issue, feel free to comment on it and we can reopen the discussion. Thanks.
cc: @hozosch, @novalis7747
